JetAnalyzers/bin/JERWriter.h

```diff
@@ -57,211 +63,323 @@ struct JERWriter
   JERWriter(const std::string& n,
 	    const std::string& e,
 	    const std::string& p,
-	    const bool b)
-    : alg(n),era(e),prefix(p),isptres(b){init();}
+	    const bool b,
+            const bool b2,
+	    const bool b3,
+        std::string s)
```

Comment:
Can we move more of these to the same line? Having them each on individual lines is silly. Just don't allow them to go beyond 120 characters per line.